src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -345,7 +348,7 @@ module.exports = function<T, P, I, TI, C, CX>(config : HostConfig<T, P, I, TI, C
       firstEffect = finishedWork.firstEffect;
     }
 
-    prepareForCommit();
+    const commitInfo = prepareForCommit();
```

Comment:
That's what I thought too but something in the DOM renderer does trigger a re-render in between `prepareForCommit` and `resetAfterCommit`, in one of the tests. I'll be honest and confess I don't know exactly why — something in the event system I'm guessing — but this seems like a less fragile solution than what we were doing previously (stashing values in global state), anyway.

If you revert the last commit you'll see the test that fails.